history/characters/570000_easter_eggs.txt

```diff
@@ -0,0 +1,293 @@
+﻿# 177000 - Greenwood
+
+570000 = { # Father of Fenarith, Arren, Derthon, and Melyina
```

Comment:
To define what is an easter egg and what is not, we had this pattern. The first easter egg character is in `wc_easteregg_characters.txt`.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/126379677-f0c8a74e-c434-457e-a8b8-3bcb27981002.png)

</details>

The easter egg dynasties are better to be named the same way.

Reply:
Fixed in 7b2b72e109bcdea81d141edeb6ad82bdba68f8e4. Relocated the characters I had added from `570000_easter_eggs.txt` to the already-existing `wc_easteregg_characters.txt` file and got rid of the now-obsolete file. Changed dynasty IDs to follow the format `easteregg_dyn<number>` to avoid confusion with character IDs.